### PR TITLE
Fix initial data for conference_auto_connect_next_speakers and datavalidator for number fields

### PR DESCRIPTION
**Pull request by jsangmeister**
`bool` is a subclass of `int` in python... WTF??

### CODE REVIEW

File: docker/initial-data.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
         "conference_stream_poster_url": "",
         "conference_open_microphone": true,
         "conference_open_video": true,
-        "conference_auto_connect_next_speakers": true,
+        "conference_auto_connect_next_speakers": 1,
```
Comment (tsiegleauq) [inline docker/initial-data.json:115]: this should initially be 0. Per default, you do not want that speakers automatically connect into jitsi

File: docs/example-data.json [hunk 2]
```diff
@@ -248,7 +248,7 @@
         "conference_stream_poster_url": "",
         "conference_open_microphone": true,
         "conference_open_video": true,
-        "conference_auto_connect_next_speakers": true,
+        "conference_auto_connect_next_speakers": 1,
```
Comment (tsiegleauq) [inline docs/example-data.json:251]: Same as above

**Review by tsiegleauq — CHANGES_REQUESTED** [review]

File: docker/initial-data.json (hunk 1 above)
Comment (jsangmeister) [reply docker/initial-data.json:115]: Ok sure, I just converted the `true` to a 1

**Review by tsiegleauq — APPROVED** [review]

File: docs/example-data.json (hunk 2 above)
Comment (FinnStutzenstein) [reply docs/example-data.json]: Quick note: We try to activate everything in the example data, so one does not have to activate everything by hand. But if this is annoying, ... it is ok to have it deactivated ;)

**Pull request merged**